modules/kernel/mkosi.build.chroot

```diff
@@ -103,11 +103,11 @@ KERNEL_RELEASE=$(make $EXTRA -s kernelrelease)
 ln -sf "$BUILDDIR" /tmp/lib
 mkdir -p "$BUILDDIR/modules/$KERNEL_RELEASE"
 find "$BUILDDIR/modules" -mindepth 1 -maxdepth 1 ! -name "$KERNEL_RELEASE" -type d -exec rm -r {} +
-make $EXTRA INSTALL_MOD_PATH=/tmp modules_install
+make $EXTRA INSTALL_MOD_PATH="$DESTDIR" MODLIB="$DESTDIR/usr/lib/modules/$KERNEL_RELEASE" modules_install
```

Comment:
This seems like a merge conflict gone wrong. This invocation should definitely always install to /tmp. However, `INSTALL_MOD_PATH` is only used in the linux kernel makefiles to initialized `MODLIB`, so if we're initializing `MODLIB` ourselves, there's no need to set `INSTALL_MOD_PATH in the first place.

Reply:
My bad. It's not a merge conflict, I saw the same `depmod` error and assumed it was caused by the same issue. The actual problem is that the SUSE `depmod` requires `/usr/lib`. I'll fix that.